tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -750,8 +751,33 @@ def _image_vector_thumb(self, filepath: Path, size: int) -> Image.Image:
             filepath (Path): The path of the file.
             size (tuple[int,int]): The size of the thumbnail.
         """
-        # TODO: Implement.
         im: Image.Image = None
+        # Create an image to draw the svg to and a painter to do the drawing
+        image: QImage = QImage(size, size, QImage.Format.Format_ARGB32)
+        image.fill("#1e1e1e")
+
+        # Create an svg renderer, then render to the painter
+        svg: QSvgRenderer = QSvgRenderer(str(filepath))
+
+        if svg.isValid():
```

Comment:
it would be nice to flip the logic here like:
```
if not svg.isValid():
    raise UnidentifiedImageError

# same code as originally except not indented
```